docs/lang/articles/contribution/contributor_guide.md

```diff
@@ -69,44 +65,43 @@ Except for minor updates, most PRs start from a developer taking over an issue.
 
 | Issue Tag                                                    | Description               | Target developer                               |
 | ------------------------------------------------------------ | ------------------------- | ---------------------------------------------- |
-| [good first issue](https://github.com/taichi-dev/taichi/issues?q=is:open+is:issue+label:"good+first+issue") |                           | Developers new to Taichi                       |
-| [welcome contribution](https://github.com/taichi-dev/taichi/issues?q=is:open+is:issue+label:"welcome+contribution") | Slightly more challenging | Developers who wish to dive deeper into Taichi |
+| [good first issue](https://github.com/taichi-dev/taichi/issues?q=is:open+is:issue+label:"good+first+issue") | Issues that are easy to start with                          | Developers new to Taichi                       |
+| [welcome contribution](https://github.com/taichi-dev/taichi/issues?q=is:open+is:issue+label:"welcome+contribution") | *Slightly* more challenging | Developers who wish to dive deeper into Taichi |
```

Comment:
```suggestion
| [welcome contribution](https://github.com/taichi-dev/taichi/issues?q=is:open+is:issue+label:"welcome+contribution") | *Slightly* more challenging issues | Developers who wish to dive deeper into Taichi |
```

Shall we consistently use noun phrases in the same column?

Reply:
Nope, it is not mandatory.